charts/datadog/values.yaml

```diff
@@ -752,6 +752,27 @@ datadog:
         # datadog.securityAgent.runtime.securityProfile.enabled -- Set to true to enable CWS runtime anomaly detection
         enabled: false
```

Comment:
based on the changelog, is this supposed to be changed to `true` ?

Reply:
I updated the PR